.vscode/tasks.json

```diff
@@ -47,35 +47,7 @@
         "lint"
       ],
       "problemMatcher": [
-        {
-          // linting errors - relative paths
-          "owner": "tslint",
-          "fileLocation": [
-            "relative",
-            "${workspaceRoot}"
-          ],
-          "severity": "warning",
-          "pattern": {
-            "regexp": "^([^/]\\S.*)\\[(\\d+), (\\d+)\\]:\\s+(.*)$",
-            "file": 1,
-            "line": 2,
-            "column": 3,
-            "message": 4
-          }
-        },
-        {
-          // linting errors - absolute paths
-          "owner": "tslint",
-          "fileLocation": "absolute",
-          "severity": "warning",
-          "pattern": {
-            "regexp": "^(/\\S.*)\\[(\\d+), (\\d+)\\]:\\s+(.*)$",
-            "file": 1,
-            "line": 2,
-            "column": 3,
-            "message": 4
-          }
-        },
+        "$tslint5",
```

Comment:
For my knowledge, can you clarify the benefits of using `tslint5` over the previous matcher algorithm?

Reply:
Well the previous matcher was not working for me. I suspect it was based on `tslint4` format. I could not be bothered to figure out the right regular expressions and just picked `tslint5` because it seems to work well.

Reply:
I decided to move this commit to a standalone PR to get it landed faster. See https://github.com/strongloop/loopback-next/pull/2162